### ISSUE TITLE
statistics is still public

### ISSUE THREAD

**Issue by sipelepis** [DSpace/DSpace#8569]
**Describe the bug**
According to the documentation, setting the property called google-analytics.authorization.admin.usage to _true_ will hide the statistics for unauthorized users. However, statistics is still visible in the home page and returns the total views for each item even though each item is inaccessible by anonymous group.

backend: DSpace 7.4
frontend:  DSpace 7.4

**To Reproduce**
1. sudo nano /dspace/config/modules/google-analytics.cfg
2. set google-analytics.authorization.admin.usage to _true_ 
3. restart tomcat, solr apache2, and postgresql.
4. start the dev environment of frontend.
5. click statistics as anonymous group user

**Expected behavior**
Statistics button and route should not be accessible on the end of the anonymous users.

**Comment by sipelepis** [comment]
Bit of context to what we did:

We've populated our website with communities, collections, and items which are previously publicly accessible. Then after populating, my team decided to set the statistics section to only be visible by authorized users such as Administrator group by setting the google-analytics.authorization.admin.usage to true and setting the authorizations for communities,collections, and items to only be accessed by admin group.

However, site statistics is still accessible by the anonymous users and returns the "total views" for each items but does not return the titles of each particular item.

**Comment by tdonohue** [comment]
@sipelepis : The statistics visible in DSpace are not driven by Google Analytics, so the `google-analytics.cfg` settings do not impact the Statistics you see in DSpace...they only impact statistics *visible in Google Analytics*... and they aren't yet supported in DSpace 7, see this note: https://github.com/DSpace/DSpace/blob/main/dspace/config/modules/google-analytics.cfg#L8

The Statistics visible in DSpace's UI are the Solr Statistics feature: https://wiki.lyrasis.org/display/DSDOC7x/SOLR+Statistics  The correct configuration you want to use is in usage-statistics.cfg:
```
usage-statistics.authorization.admin.usage = true
```

I've tested locally and verified that this configuration works.  You were just using the wrong setting.  Closing this ticket as answered.
